doc/tutorials/specification.md

```diff
@@ -45,7 +45,7 @@ Make sure you have `Elektra` installed on your local machine:
 ```
 kdb --version
 
-KDB_VERSION: 0.9.6
+KDB_VERSION: 0.9.9
```

Comment:
As we don't depend on any particular feature of this version, I think we can remove this part.

Reply:
Do you want me to remove the change from 0.9.6 to 0.9.9, or the complete call of `kdb --version`?

Reply:
Unfortunately GitHub does not allow me to make a suggestion (not allowed on deleted lines). I would remove the whole output of `kdb --version`.

Reply:
This was not the suggestion I wanted to make. I wanted to remove the whole block and GitHub explicitly said that this kind of suggestion is not supported.

Reply:
Ah yes, that is very interesting. You can select both the

```
    ```
    kdb --version
```

and the 

```
    ```
```

lines for suggestions, but you cannot select all of them, because of the deleted line in between. Seems like a very odd limitation of GitHub...